scipy/stats/tests/test_stats.py

```diff
@@ -403,137 +403,6 @@ def test_len1(self):
         assert_raises(ValueError, stats.pearsonr, x, y)
 
 
-class TestFisherExact(object):
-    """Some tests to show that fisher_exact() works correctly.
-
-    Note that in SciPy 0.9.0 this was not working well for large numbers due to
-    inaccuracy of the hypergeom distribution (see #1218). Fixed now.
-
-    Also note that R and SciPy have different argument formats for their
-    hypergeometric distribution functions.
-
-    R:
-    > phyper(18999, 99000, 110000, 39000, lower.tail = FALSE)
-    [1] 1.701815e-09
-    """
-
-    def test_basic(self):
```

Comment:
Can you explain where these and `test_precise` went?

Reply:
The test class `TestFisherExact` should cover all these matrices.  Note that the results from R are stored in the generated file `fisher_exact_results_from_r.py`.  This data is used in the test method `TestFisherExact.test_results_from_r` in `test_fisher_exact.py`.